platform/app-manager/app-manager.js

```diff
@@ -18,71 +21,109 @@ mongoClient.connect(mongoUrl, { useNewUrlParser: true }, function(err, client) {
     db = client.db(appsDb);
 });
 
-// Create logs directory if not present
+// Create logs directory for apps if not present
 fs.ensureDirSync(`${__dirname}/logs`);
 /**
```

Comment:
Add TODO: When app manager starts, check in DB what apps were running and then restart those apps. If there are zombie processes, kill those (use pid).